vscode/src/client/types.ts

```diff
@@ -57,8 +57,8 @@ export interface KaiRpcApplicationConfig {
 
   analyzerLspLspPath: string;
   analyzerLspRpcPath: string;
-  analyzerLspRulesPath: string; // TODO: See https://github.com/konveyor/kai/issues/509
-  analyzerLspJavaBundlePath: string; // TODO: See https://github.com/konveyor/kai/issues/547
+  analyzerLspRulesPath: string[];
```

Comment:
I was just seeing this with latest. You probably need to update the names to Paths so they are passed correctly